wpilib/wpilib/driverstation.py

```diff
@@ -657,7 +657,7 @@ def _getData(self):
             hal.getJoystickPOVs(stick, self.joystickPOVsCache[stick])
             hal.getJoystickButtons(stick, self.joystickButtonsCache[stick])
 
-        hal.getMatchInfo(self.matchInfoCache)
+        self.matchInfoCahce = hal.MatchInfo()
```

Comment:
This is misspelled, and it's no longer calling getMatchInfo.

Reply:
https://github.com/robotpy/robotpy-wpilib/pull/414/files#diff-cffd12ee377c3edf8d638d6ca2cf25c5R87

I may have jumped the gun on functionality here. Should these not operate the same?